src/main/java/sirius/biz/tenants/TenantData.java

```diff
@@ -218,7 +218,7 @@
      * Contains the package, upgrades and additional and revoked permissions for this tenant.
      */
     public static final Mapping PACKAGE_DATA = Mapping.named("packageData");
-    private final PackageData packageData = new PackageData();
```

Comment:
why not initialize in place?

Reply:
because `this` is not the tenant, only TenantData